src/codec/length.rs

```diff
@@ -1,5 +1,6 @@
 use crate::{Decoder, Encoder};
-use bytes::{BigEndian, BufMut, ByteOrder, Bytes, BytesMut};
+use bytes::{Bytes, BytesMut, Buf, BufMut};
+use byteorder::{BigEndian, ByteOrder};
```

Comment:
it looks like this isn't used?

Reply:
Sorry for just seeing this comment now! I am not sure what you mean here - what exactly is not being used?

Reply:
The `BigEndian` and `ByteOrder` imports from the `byteorder` crate—i think that entire dependency can be removed? :thinking:

Reply:
Ok, I think I see what you mean. Pushing changes now - please let me know if this is what you had in mind. Thanks! Alex